common/display/display-en.html

```diff
@@ -0,0 +1,970 @@
+	<li><a href="#col-count">Col count</a></li>
+	<li><a href="#sizing-controls">Sizing controls</a></li>
+</ul>
+
+<h2 id="bootstrap">Bootstrap</h2>
+
+<h3 id="float">Float</h3>
+
+<h4>Bootstrap 3 <code> .clearfix</code>working example</h4>
+<div class="well">
+	<div class="row">
+		<div class="col-md-3 well" style="background-color: #ffffff; float: left;">Column 1</div>
+		<div class="col-md-3 well" style="background-color: #ffffff; float: left;">Column 2</div>
+		<div class="col-md-3 well" style="background-color: #ffffff; float: left;">Column 3</div>
+		<!-- Add clearfix class to clear floated elements -->
+		<div class="clearfix"></div>
+	</div>
+	<p class="well">This is some content below the floated columns. Without clearfix, this content would overlap with the floated columns. But clearfix ensures proper layout by clearing the floats.</p>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="row"&gt;
+	&lt;div class="col-md-3 well" style="background-color: #ffffff; float: left;"&gt;Column 1&lt;/div&gt;
+	&lt;div class="col-md-3 well" style="background-color: #ffffff; float: left;"&gt;Column 2&lt;/div&gt;
+	&lt;div class="col-md-3 well" style="background-color: #ffffff; float: left;"&gt;Column 3&lt;/div&gt;
+	&lt;!-- Add clearfix class to clear floated elements --&gt;
+	&lt;div class="<strong>clearfix</strong>"&gt;&lt;/div&gt;
+&lt;/div&gt;
+&lt;p class="well"&gt;This is some content below the floated columns. Without clearfix, this content would overlap with the floated columns. But clearfix ensures proper layout by clearing the floats.&lt;/p&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>Bootstrap 3 <code>.center-block</code> class working example</h4>
+<div class="well">
+	<div class="center-block well" style="width: 200px; height: 200px; background-color: #ffffff;">
+		This element is centered horizontally within its parent container.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>center-block</strong> well" style="width: 200px; height: 200px; background-color: #ffffff;"&gt;
+		This element is centered horizontally within its parent container.
+&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>Bootstrap 3 <code>.pull-right</code> class working example</h4>
+<div class="well" style="height: 115px;">
+	<div class="pull-right well">
+		This element is floated to the right within its parent container.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well" style="height: 115px;"&gt;
+	&lt;div class="<strong>pull-right</strong> well"&gt;
+		This element is floated to the right within its parent container.
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>Bootstrap 3 <code>.pull-left</code> class working example</h4>
+<div class="well" style="height: 115px;">
+	<div class="pull-left well">
+		This element is floated to the left within its parent container.
+	</div>
+</div>
+<div class="clearfix"></div>
+&lt;div class="clearfix"&gt; &lt;/div&gt;
```

Comment:
Is this necessary here? the pull-right example don't seem to need that

```suggestion
```

Reply:
Fixed